### DISCUSSION TITLE
How to post new schema via API

### DISCUSSION THREAD

**Discussion by dtseng123** [openownership/data-standard#481] [Q&A] [answered]
I can see how I would manually send a submission. https://register.openownership.org/submissions But, how would I programmatically do so? I was expecting the ability to post new submissions via a REST API. Is this on the roadmap?

**Comment by StephenAbbott** [comment, marked as answer]
Hi @dtseng123. Thanks for the question. The user submissions feature - https://register.openownership.org/submissions - for the Open Ownership Register was implemented several years ago and has not seen the uptake we'd hoped. 

As part of our ongoing work to update the Register to publish data in line with version 0.2 of our data standard and to make other improvements (see https://github.com/openownership/register-v2), we will be removing the user submissions feature and instead redoubling our efforts on combining official data published by national beneficial ownership registers.

Our team at Open Ownership currently does not have the capacity to verify or check user-submitted beneficial ownership data so we think it best to focus on helping people to use national level datasets instead.

**Comment by dtseng123** [comment]
Thanks, @StephenAbbott. The user submission feature wouldn't see the uptake if there is no way to upload new schemas via JSON or API.

There's little to no benefit if the user experience of creating one is manual. Case in point: I was planning on connecting submissions from my company and platform but that seems like it is a non-starter now. 

Have you all considered a Wikipedia type review feature for users to cross-review submissions?
